### ISSUE TITLE
Broken packge state?

### ISSUE THREAD

**Issue by vtolstov** [hashicorp/go-getter#25]
```
go-getter git::https://github.com/yoctocloud/packer-centos.git ./tests
2016/04/29 18:39:01 Error downloading: /usr/host/bin/git exited with 128: Cloning into '/tmp/getter-git175513477'...
fatal: repository 'https://github.com/yoctocloud/' not found
```

why command strips last component?
i'm try use go-getter lib in my app and seems that it strips last component too.

**Comment by vtolstov** [comment]
As i see mode Any thinks in this case that this is file packer-centos with ext git.
Why this happened ? I'm specify protocol git....

**Comment by ryanuber** [comment]
@vtolstov I think this should be resolved after #41 above, so I'm going to close this. Let me know if you get a chance to try it out or if you still see this issue when building from master. Thanks!
